demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -111,8 +111,11 @@ def from_path(path: Path) -> Optional["BaseContent"]:
         logger.info(f"Loading content item from path: {path}")
         if path.is_dir() and path.parent.name == "Packs":  # if the path given is a pack
             return content_type_to_model[ContentType.PACK].from_orm(PackParser(path))
-        content_item_parser = ContentItemParser.from_path(path)
-
+        try:
+            content_item_parser = ContentItemParser.from_path(path)
+        except Exception as e:
+            logger.error(f"Could not parse content item from path: {path}. Error: {e}")
```

Comment:
what kinds of errors do we except here? Perhaps we should raise it, to stop higher-level scripts from running with `None` values?

Reply:
If the path given is not a content item, an exception could be raised.
Maybe it's worth to raise Excpetion on all invalid cases, and never return None, but thath's for a different PR.